docs.html

```diff
@@ -758,6 +760,113 @@ <h3>Demo</h3>
+                            <label for="rate-4" class="fa fa-star rating-label"></label>
+                            <input type="radio" id="rate-3" class="rating-radio">
+                            <label for="rate-3" class="fa fa-star rating-label"></label>
+                            <input type="radio" id="rate-2" class="rating-radio">
+                            <label for="rate-2" class="fa fa-star rating-label"></label>
+                            <input type="radio" id="rate-1" class="rating-radio">
+                            <label for="rate-1" class="fa fa-star rating-label"></label>
+                       </div>
+                    </div>
+                    <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-dark&wt=none&l=htmlmixed&width=724&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%250A%253Cdiv%2520class%253D%2522interactive-rating%2522%253E%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rate-5%2522%2520class%253D%2522rating-radio%2522%253E%250A%2520%2520%253Clabel%2520for%253D%2522rate-5%2522%2520class%253D%2522fa%2520fa-star%2520rating-label%2522%253E%253C%252Flabel%253E%250A%2520%2520%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rate-4%2522%2520class%253D%2522rating-radio%2522%253E%250A%2520%2520%253Clabel%2520for%253D%2522rate-4%2522%2520class%253D%2522fa%2520fa-star%2520rating-label%2522%253E%253C%252Flabel%253E%250A%2520%2520%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rate-3%2522%2520class%253D%2522rating-radio%2522%253E%250A%2520%2520%253Clabel%2520for%253D%2522rate-3%2522%2520class%253D%2522fa%2520fa-star%2520rating-label%2522%253E%253C%252Flabel%253E%250A%2520%2520%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rate-2%2522%2520class%253D%2522rating-radio%2522%253E%250A%2520%2520%253Clabel%2520for%253D%2522rate-2%2522%2520class%253D%2522fa%2520fa-star%2520rating-label%2522%253E%253C%252Flabel%253E%250A%2520%2520%250A%2520%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522rate-1%2522%2520class%253D%2522rating-radio%2522%253E%250A%2520%2520%253Clabel%2520for%253D%2522rate-1%2522%2520class%253D%2522fa%2520fa-star%2520rating-label%2522%253E%253C%252Flabel%253E%250A%253C%252Fdiv%253E%250A"
+                    style="width: 742px; height: 505px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </section>
+
+            <!--Toast Component-->
+            <section id="toast" class="component-section">
+                <div class="component-container">
+                    <h2 class="component-heading">Toast</h2>
+                    <p class="component-description">

```

Comment:
LGTM , just one thing to make your HTML code consistent, in rating component description yu have used <span> , and in toast <p> , use only one to keep consistency

Reply:
Sure will keep in mind and make the changes. Thank you.